src/public/modules/forms/services/form-logic.client.service.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  BasicField,
+  LogicConditionState,
+  LogicValidConditions,
+} from '../../../../types'
+
+const conditions: LogicValidConditions[] = [
```

Comment:
suggest renaming this to `VALID_CONDITIONS`

Reply:
ok